src/helpers/cities.ts

```diff
@@ -1,27 +1,113 @@
-export const popularCities = [ 
-  {
-    name: "Denver" ,
-    lat: 39.77,
-    long: -105.00,
-  },
-  {
-    name: "San Francisco",
-    lat: 37.76,
-    long: -122.39,
-  },
-  {
-    name: "Miami",
-    lat: 25.79,
-    long: -80.21,
-  },
-  {
-    name: "Houston",
-    lat: 29.90,
-    long: -95.32,
-  },
-  {
-    name: "New York City",
-    lat: 40.69,
-    long: -73.92,
-  }
-]
+export const popCities = [
```

Comment:
I like that we have hardcoded data to display while we style and manage our app before deployment. I do also agree that we cannot hard code in weather data! So I'm not opposed to making the fetch calls for the cards upon page load and updating that as we wrap up!